addon/components/citation-widget/template.hbs

```diff
@@ -1,11 +1,16 @@
 <div>
     <div id="citationList" class="m-b-md">
-        <div class="f-w-xl">{{t 'eosf.components.citationWidget.apa'}}</div>
-        <span> {{apa}} </span>
-        <div class="f-w-xl m-t-md">{{t 'eosf.components.citationWidget.mla'}}</div>
-        <span> {{mla}} </span>
-        <div class="f-w-xl m-t-md">{{t 'eosf.components.citationWidget.chicago'}}</div>
-        <span> {{chicago}} </span>
+        {{#each provider.citationStyles as |style|}}
+            <div class="f-w-xl">{{style.title}}</div>
```

Comment:
Do you not need `m-t-md` on the non-first elements? Or does it look fine without?

Reply:
It's a little cramped, I guess, but not too bad.

Reply:
I missed this comment! I will add a `m-t-md` for the non-0th one